messenger.py

```diff
@@ -50,9 +50,9 @@ def serialize(self):
             'oid': self.oid})
 
 
-class Messenger(ExportedGObject):
+class Messenger(CollabWrapper):
```

Comment:
@Creatune I don't understand these changes are these changes required? Also, the `telepathy` port seems incomplete. 
The `activity.py` file requires more changes.

Reply:
I will have to learn how collabwrapper works. For now, what I did was just a guess. I will get back to that later :)